.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Continuous Integration
+
+on: [push]
+
+jobs:
+  test:
+    name: Test by running cookiecutter
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: 'orbyter-cookiecutter'
+      - name: Install cookiecutter
+        run: pip install cookiecutter
+      - name: run ls
+        run: ls
+      - name: Cut a cookie (repo)
+        run: python -m cookiecutter --no-input orbyter-cookiecutter project_name=foundation repo_name=foundation
+      - name: Build project containers (make dev-start)
+        working-directory: foundation
```

Comment:
This `workig-directory` is set the value value in a few places. Not necessary, but you could parametrize this. Adding link in case you're interested: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter

Reply:
You can also do it with env vars https://docs.github.com/en/actions/reference/environment-variables